src/main/resources/i18n/messagges_it_IT.properties

```diff
@@ -0,0 +1,284 @@
+EMPTY_LABEL=
```

Comment:
please rename the file as messages_it_IT.properties.

there is an extra `g`

Reply:
Done, sorry